scan/nogit.go

```diff
@@ -73,6 +74,27 @@ func (ngs *NoGitScanner) Scan() (Report, error) {
 			}
 
 			for _, rule := range ngs.cfg.Rules {
+				if rule.Multiline {
+					bytesRead, _ := ioutil.ReadFile(p)
```

Comment:
Shouldn't we check the error here? Files which are too large to fit in 1 operation will silently fail.